### PR TITLE
Update markdown files

### PR DESCRIPTION
**Pull request by jdhoffa**
This is a pretty minor PR that brings this repo up-to-date, and fixes a few style thing (no content changes). Mainly I just want to bring a bit of attention back to this repo, as we try to document our best practices for working together (both technical practices, and human interaction practices). 

As a reminder, the practices render in a gh-pages site here: https://rmi-pacta.github.io/practices/

Concretely in this PR, I:
- Update some refs in README
- Remove an excessive Rmd structure
- fix style and grammar in git.md

### CODE REVIEW

**Comment by jdhoffa** [comment]
Requesting reviews from the entire tech team, because in general I think PRs to this repo should be made by consensus.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+This document intends to document how we develop software at RMI, including
 social and technical aspects around a number of themes. It should help guide
 our actions,  and understand what to expect from our colleagues and what they
 expect from us.  Wherever possible, we refer to the source of information
 that defines a "best practice". Anecdotes and personal experience have
 value, but we prefer evidence from research, books, and large and mature
 communities of practice.
 
-We have agreed to this practices by consensus. They document heuristics, as
+We have agreed to these practices by consensus. They document heuristics, as
```
Comment (cjyetman) [inline README.md:11]: did we? LOL

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 detailed instructions may never cover all scenarios. We focus on _what_ to do
 and _why_, and only recommend optional resources about _how_ to do it.
 We realize there may be times when you can not achieve all of these practices.
 In those cases, it's nice to let others know, e.g. in a commit description or a
 PR comment, how or why some of these guidelines could not be met.
 
 * [Google Doc](https://bit.ly/2dii-practices)
```
Comment (cjyetman) [inline README.md:18]: this is still 2dii specific

File: git.md [hunk 3]
```diff
@@ -11,158 +9,94 @@ commits locally).
...
-explain the motivation for the change (“why”, not “what” or “how”).
+Like an email, commit messages have a subject and an optional body. The subject should start with a line summarizing your changes (aim for 50 characters or less). Use the imperative, present tense – for example, “Fix typo”, not “Fixed typo” or “Fixes typo”. The message body, if present, should follow a blank line; wrap around 70 characters; and explain the motivation for the change (“why”, not “what” or “how”).
 
 See also:
 
--   [Example of a commit
-    message](https://github.com/2DegreesInvesting/resources/issues/74).
--   [Commit best
-    practices](https://r-pkgs.org/git.html#commit-best-practices).
+-   [Example of a commit message](https://github.com/2DegreesInvesting/resources/issues/74).
```
Comment (cjyetman) [inline git.md]: 2dii specific

File: git.md [hunk 4]
```diff
@@ -11,158 +9,94 @@ commits locally).
...
-you choose to work, just make sure to agree on a common workflow that
-everyone follows:
+Choose a workflow based on your team’s needs and preferences. However you choose to work, just make sure to agree on a common workflow that everyone follows:
 
 GitHub workflow:
 
-This workflow is 2DII’s default. If a project uses a different workflow,
-document prominently (e.g. in README) what that workflow is
-([example](https://github.com/github/gitignore#contributing-workflow)).
+This workflow is 2DII’s default. If a project uses a different workflow, document prominently (e.g. in README) what that workflow is ([example](https://github.com/github/gitignore#contributing-workflow)).
```
Comment (cjyetman) [inline git.md]: 2dii specific

File: git.md [hunk 5]
```diff
@@ -11,158 +9,94 @@ commits locally).
...
-[Version control best
-practices](https://www.git-tower.com/blog/version-control-best-practices/)
+[Version control best practices](https://www.git-tower.com/blog/version-control-best-practices/)
 
 ## How
 
 -   [Git practices by example](TODO)
--   [Coding
-    helpdesk](https://github.com/2DegreesInvesting/coding-helpdesk#coding-helpdesk).
+-   [Coding helpdesk](https://github.com/2DegreesInvesting/coding-helpdesk#coding-helpdesk).
```
Comment (cjyetman) [inline git.md]: 2dii specific

**Review by cjyetman — DISMISSED** [review]
still a handful of 2dii specific stuff, but fine by me if you want to commit this now and deal with those later

File: README.md (hunk 1 above)
Comment (jdhoffa) [reply README.md:11]: I mean, in principle, we agreed on these 3 years ago hahahah. Definitely up for interpretation if we still agree to this, but that's exactly why I'm bringing attention back to this repo :-)

File: README.md (hunk 2 above)
Comment (jdhoffa) [reply README.md:18]: I think this is ok, it's just the URL. The document is still useful (and if we end up making our own document, we can link it in the future)

File: git.md (hunk 3 above)
Comment (jdhoffa) [reply git.md]: Updated

File: git.md (hunk 4 above)
Comment (jdhoffa) [reply git.md]: Updated

File: git.md (hunk 5 above)
Comment (jdhoffa) [reply git.md]: I am just going to review this entirely, since we don't really offer anything like a coding helpdesk anymore

**Comment by jdhoffa** [comment]
Ok, comments addressed :-)

**Review by cjyetman — APPROVED** [review]

**Pull request merged**